embeddings/evaluator/evaluation_results.py

```diff
@@ -47,22 +58,9 @@ def metrics(self) -> Dict[str, Any]:
 
 @dataclass
 class TextClassificationEvaluationResults(EvaluationResults):
-    accuracy: float
-    f1_macro: float
-    f1_micro: float
-    f1_weighted: float
-    recall_macro: float
-    recall_micro: float
-    recall_weighted: float
-    precision_macro: float
-    precision_micro: float
-    precision_weighted: float
+    pass
 
 
 @dataclass
 class SequenceLabelingEvaluationResults(EvaluationResults):
```

Comment:
I left it in case if it will be used in future. Revert of removing it would require much work. We can remove it when the library will be more stable ;)